src/providers/router/index.tsx

```diff
@@ -17,19 +17,19 @@ export const router = createBrowserRouter([
             }
         ]
     },
-    
+
     {
         path: "/",
         element: <GuestLayout/>,
         children:[
             {
-                path: "/",
+                path: "/login",
```

Comment:
Вынеси все `path` в константу `ROUTES`
```
const ROUTES = [
  LOGIN: {
     path: '/your',
  }
}
```

```suggestion
                path: ROTES.LOGIN.path,
```

И т.д.

Reply:
исправлено